addons/account/models/chart_template.py

```diff
@@ -641,8 +646,15 @@ def _setup_utility_bank_accounts(self, template_code, company, template_data):
         for company_attr_name, account in zip(accounts_data.keys(), accounts):
             company[company_attr_name] = account
 
+    def _add_foreign_taxes_data(self, company, countries, data):
+        for country in countries:
+            country_data = self._get_foreign_chart_template_data(country, company, reload=True)
+            for model in ('account.tax', 'account.tax.group'):
+                data[model].update(country_data[model])
+        return data
+
     @api.model
-    def _instantiate_foreign_taxes(self, country, company):
+    def _get_foreign_chart_template_data(self, country, company, reload=False):
```

Comment:
Some problems I noticed here:

- If we modify the method, we should also update the docstring.
It's not creating taxes anymore.

- The method name here is "get" but the method still creates accounts through the `create_foreign_tax_account` inner function.
- I saw the code will come twice in this function (during `_load` and `_load_translations`) which means that part of the method that creates accounts without even checking if there are already some will create some accounts twice.

To avoid that, we could:
- create an `account.account` dict and make those accounts creation being done by the same like for the others
- just put their xml_id reference in taxes and let the system retrieve and link them